.gitignore

```diff
@@ -38,6 +38,7 @@ lib/generated_plugin_registrant.dart
 
 # Node js app
 node_server/node_modules
+node_server/report.20200510.102359.3840.0.001.json
```

Comment:
Do we expect to see this exact json file in every case? If not remove this change from `package.json`.

Reply:
Ok i would took care of it in the next commit.